admin_guide/cluster_management.rst

```diff
@@ -54,49 +54,11 @@ In this sub-section, we discuss how you can deal with node failures without incu
+Citus can easily tolerate worker node failures because of its logical sharding-based architecture. Each worker work in the cluster can have standbys in case of failure. Worker replication is controlled in one of two ways depending on the circumstances.
 
-::
-
-    WARNING:  could not connect to node localhost:9700
-
-On seeing such warnings, the first step would be to log into the failed node and
-inspect the cause of the failure.
-
-In the meanwhile, Citus will automatically re-route the work to the healthy workers. Also, if Citus is not able to connect to a worker, it will assign that task to another node having a copy of that shard. If the failure occurs mid-query, Citus does not re-run the whole query but assigns only the failed query fragments leading to faster responses in face of failures.
-
-Once the node is brought back up, Citus will automatically continue connecting to it and
-using the data. 
-
-If the node suffers a permanent failure then you may wish to retain the same
-level of replication so that your application can tolerate more failures. To
-make this simpler, Citus enterprise provides a replicate_table_shards UDF which
-can be called after. This function copies the shards of a table across the
-healthy nodes so they all reach the configured replication factor.
-
-To remove a permanently failed node from the list of workers, you should first
-mark all shard placements on that node as invalid (if they are not already so)
-using the following query:
-
-::
-
-   UPDATE pg_dist_shard_placement set shardstate = 3 where nodename = 'bad-node-name' and nodeport = 5432;
-
-Then, you can remove the node using master_remove_node, as shown below:
-
-::
-   
-   select master_remove_node('bad-node-name', 5432);
-
-If you want to add a new node to the cluster to replace the
-failed node, you can follow the instructions described in the
-:ref:`adding_worker_node` section. Finally, you can use the function provided in
-Citus Enterprise to replicate the invalid shards and maintain the replication factor.
-
-::
-
-    select replicate_table_shards('github_events');
+1. **PostgreSQL streaming replication.** This option is best for heavy OLTP workloads. It replicates entire worker databases by continuously streaming their WAL records to a standby. You can configure streaming replication on-premise yourself by consulting the `PostgreSQL replication documentation <https://www.postgresql.org/docs/current/static/warm-standby.html#STREAMING-REPLICATION>`_ or use :ref:`Citus Cloud <cloud>` which is pre-configured for replication and high-availability.
 
+2. **Citus shard replication.** This option suffices for an append-only workload. It duplicates shards between workers, automatically marking shards unavailable in master node records as necessary and using replicas to serve your queries. To enable shard replication simply set :code:`SET citus.shard_replication_factor = 2;` (or higher) before distributing data to the cluster.
```

Comment:
Maybe instead of 'suffices', something like 'best suited for'? 

> It duplicates shards between workers, automatically marking shards unavailable in master node records as necessary and using replicas to serve your queries.

Maybe instead of diving straight into marking shards as invalid, let's give a brief description? Something like: "Citus replicates shards across different nodes by automatically replicating DML statements and managing consistency. If a node goes down, the co-ordinator node will continue to serve queries by routing the work to the replicas seamlessly"